src/unstract/sdk/adapters/embedding/open_ai/src/open_ai.py

```diff
@@ -54,13 +56,17 @@ def get_json_schema() -> str:
     def get_embedding_instance(self) -> BaseEmbedding:
         try:
             timeout = int(self.config.get(Constants.TIMEOUT, Constants.DEFAULT_TIMEOUT))
+            httpx_timeout = httpx.Timeout(10.0, connect=60.0)
+            httpx_client = httpx.Client(timeout=httpx_timeout)
             embedding: BaseEmbedding = OpenAIEmbedding(
                 api_key=str(self.config.get(Constants.API_KEY)),
                 api_base=str(
                     self.config.get(Constants.API_BASE_KEY, Constants.API_BASE_VALUE)
                 ),
+                model=str(self.config.get(Constants.MODEL)),
```

Comment:
@gaya3-zipstack the default value will be `None` for existing projects.
Either add a migration for this or specify a default here